lms/static/sass/views/_bookmarks.scss

```diff
@@ -0,0 +1,141 @@
+// Rules for placing bookmarks and search button side by side
+.wrapper-course-modes {
+  border-bottom: 1px solid lightgray;
```

Comment:
NIT: please use one of the gray variables, such as "$gray-l4" (I don't know if that's the one that is closest to what you have, but I'm sure one would work here.